lib/htmlbeautifier/builder.rb

```diff
@@ -66,7 +66,7 @@ def embed(opening, code, closing)
 
     def foreign_block(opening, code, closing)
       emit opening
-      unless code.empty?
+      unless code.empty? || code.match(/\A\s+\z/)
```

Comment:
We can also use `code.strip.empty?` here. It's terser and a bit clearer, I think.